TooltipDecorator/TooltipDecorator.js

```diff
@@ -425,6 +442,7 @@ const TooltipDecorator = hoc(defaultConfig, (config, Wrapped) => {
 						{...tooltipProps}
 						arrowAnchor={this.state.arrowAnchor}
 						direction={this.state.tooltipDirection}
+						marquee={tooltipMarquee}
 						position={tooltipRelative ? null : this.state.position}
 						relative={tooltipRelative}
 						style={{display: ((tooltipRelative && !this.state.showing) ? 'none' : null)}}
```

Comment:
I've noticed we didn't sync up some of code from https://github.com/enyojs/sandstone/pull/298 which defines css variables for top and left position.
I think we need this to set different tooltip position for Progressbar and Slider.
Currently, the tootip is positioned too far from the Progressbar. Would you please consider this?

Reply:
I updated the position css variables to match sandstone.
I fixed the tooltip position for Slider. For Progressbar I set an offset of `6px` instead of `12px` (there's no specifications in Zepplin for this case). What do you think?